### DISCUSSION TITLE
quectel test on srsRAN4G (PING)

### DISCUSSION THREAD

**Discussion by initChat** [srsran/srsRAN_4G#1276] [Q&A]
I used to use UE as samsung a7 and successful with it. Right now I want to use Quectel RM510Q-GL with sysmocom sim card as UE.
Currently I can only ping using 'AT+QPING=1,"8.8.8.8" on minicom
when I run 'AT+QPING=1,"8.8.8.8"' on minicom
the screen show 
```
at+qping=1,"8.8.8.8"                                                            
OK                                                                              
 +QPING: 569   

```
and the tracer of gNB (BladeRF) showing 
```
               -----------------DL----------------|-------------------------UL-------------------------
rat  pci rnti  cqi  ri  mcs  brate   ok  nok  (%) | pusch  pucch  phr  mcs  brate   ok  nok  (%)    bsr
lte    1   62   15   1    5   2.3k    6    0   0% |  23.4   23.1   40   18    79k   11    0   0%    0.0
lte    1   62   15   1    0      0    0    0   0% |   n/a   99.9    0    0      0    0    0   0%    0.0
lte    1   62   15   1    0      0    0    0   0% |   n/a   99.9    0    0      0    0    0   0%    0.0
lte    1   62   15   1    0      0    0    0   0% |   n/a   99.9    0    0      0    0    0   0%    0.0
```

The bit rate only occur once and never occur again. But I woud like to test for longer time. Is there a command for that? for ping continuously?

However, if you guys don't use QPING command, what other command you can use, to make quectel run continuously on srsRAN?

Currently I use 
SDR: BladeRF A4
UE: Quectel RM510Q-GL with sysmocom sim card
And run srsRAN_4G with a core network from open5GS

**Comment by dhiaboujebha** [comment]
@pchat-imm Any solutions found?

**Comment by NicolasGresset** [comment]
Hi,
Kind of a late answer but It could still help you. I assume you have connected your Quectel modem to a desktop. This way, you can start a ping command directly from your desktop, e.g ping the 10.45.1.1 (IIRC the address of the CN in the subnet allocated for UE addresses).

Otherwise, if the IP allocated for your UE is 10.45.1.2, you can still ping directly from the CN. I believe you need to add a specific route for that purpose in Open5GS : 

```bash
sudo ip route add 10.45.0.0/16 via 10.53.1.2
``` 

Best,
Nicolas
